AndelaOSP/andela-societies-frontend
# Resolves #33 - Ability for users to be able to log out

Resolves #33 

Description
 This gives users the ability to log out from the application when they are signed in.

Fix
 - Add a function that deletes cookies.
 - Add a function that logs out users after cookies have been deleted successfully.

How to test
 - Pull this branch if you don't have it already. (git pull origin  ft-user-should-be-able-to-logout-155808774).
 - Run yarn start. The signIn page should appear.
 - Click on the Sign in with Google button.
 - Once you are signed in successfully, click on the user avatar that displays a drop-down.
 - Then click on the Sign Out link, it should redirect you back to the sign in page.
